public/app/themes/justice/src/sass/admin.scss

```diff
@@ -1,3 +1,49 @@
+// Styling for the PublishPress Revisions plugin
+
+// Hide the upsell menu link
+.toplevel_page_revisionary-q {
+  ul.wp-submenu-wrap {
```

Comment:
Hey @EmilyHazlehurst 😄 
Davey has put a brilliant case forward for using URL selection instead. I'm sorry for muddying the waters here with my suggestion.

I was wrong about the DOM load problem I thought we had. JS was changing the URL, causing the page's jump effect. I've enjoyed the conversation around this stuff.

Thanks to both you and @EarthlingDavey for the opportunity 👍

Reply:
Thanks @wilson1000 @EarthlingDavey, I think that should be everything but let me know if there's anything else :)